File: android/versioned-abis/expoview-abi39_0_0/src/main/java/abi39_0_0/expo/modules/application/ApplicationPackage.java
package abi39_0_0.expo.modules.application;

import android.content.Context;

import java.util.Collections;
import java.util.List;

import abi39_0_0.org.unimodules.core.BasePackage;
import abi39_0_0.org.unimodules.core.ExportedModule;
import abi39_0_0.org.unimodules.core.ViewManager;

public class ApplicationPackage extends BasePackage {
  @Override
  public List<ExportedModule> createExportedModules(Context context) {
    return Collections.singletonList((ExportedModule) new ApplicationModule(context));
  }
}

File: packages/expo-application/android/src/main/java/expo/modules/application/ApplicationPackage.kt
package expo.modules.application

import android.content.Context
import expo.modules.core.BasePackage
import expo.modules.core.ExportedModule

class ApplicationPackage : BasePackage() {
  override fun createExportedModules(context: Context): List<ExportedModule> {
    return listOf(ApplicationModule(context) as ExportedModule)
  }
}
